sentry-samples/sentry-samples-spring-boot/src/main/java/io/sentry/samples/spring/boot/CustomEventProcessor.java

```diff
@@ -0,0 +1,35 @@
+package io.sentry.samples.spring.boot;
+
+import io.sentry.core.EventProcessor;
+import io.sentry.core.SentryEvent;
+import io.sentry.core.protocol.SentryRuntime;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.stereotype.Component;
+
+/**
+ * Custom {@link EventProcessor} implementation lets modifying {@link SentryEvent}s before they are
+ * sent to Sentry.
+ */
+@Component
+public class CustomEventProcessor implements EventProcessor {
+  private final String javaVersion;
+  private final String javaVendor;
+
+  public CustomEventProcessor(String javaVersion, String javaVendor) {
+    this.javaVersion = javaVersion;
+    this.javaVendor = javaVendor;
+  }
+
+  public CustomEventProcessor() {
+    this(System.getProperty("java.version"), System.getProperty("java.vendor"));
+  }
+
+  @Override
+  public SentryEvent process(SentryEvent event, @Nullable Object hint) {
+    final SentryRuntime runtime = new SentryRuntime();
```

Comment:
Would be nice to do this to all events (put it in the main event processor)
Likely not going to work for Android though

Reply:
yeah it won't, we could have the processor on sentry-core but either it gets added to the processor list only if its non-Android or we do a conditional flag.